packages/cloud_firestore/cloud_firestore/example/integration_test/vector_value_e2e.dart

```diff
@@ -0,0 +1,172 @@
+// Copyright 2020, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void runVectorValueTests() {
+  group('$VectorValue', () {
+    late FirebaseFirestore firestore;
+
+    setUpAll(() async {
+      firestore = FirebaseFirestore.instance;
+    });
+
+    Future<DocumentReference<Map<String, dynamic>>> initializeTest(
+      String path,
+    ) async {
+      String prefixedPath = 'flutter-tests/$path';
+      await firestore.doc(prefixedPath).delete();
+      return firestore.doc(prefixedPath);
+    }
+
+    testWidgets('sets a $VectorValue & returns one', (_) async {
```

Comment:
Nice work updating the tests 🔥! Could we change testWidgets() to test() instead?